src/changelog/2023/11/project-nodes-devices.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Project Nodes for Devices
+date: 2023-11-23 11:02:00.0
+authors: ["marian-demme"]
+tags:
+    - changelog
+---
+
+ From this update onwards, Project Nodes will now be available for devices assigned to an application. Previously, it was necessary for devices to be bound to an instance to utilize this functionality.
```

Comment:
Explain the difference between an instance and an application please

Reply:
[difference between Instances, Devices, and Applications](https://github.com/FlowFuse/website/pull/1338/commits/01826c568eb45000c9337a430fce6d349cb5e7b9)